pkg/config/type.go

```diff
@@ -51,6 +51,8 @@ const (
 	OSWindows = "windows"
 	// OSLinux is the the linux Operating System
 	OSLinux = "linux"
+	// Node termination finalizer on CNINode CRD
+	NodeTerminationFinalizer = "node.k8s.aws.com/termination"
```

Comment:
Let's discuss on the naming since this will be very hard to change in the future if we have to.

Reply:
Probably `networking.k8s.aws/cninode`? For example, the finalizer key used by network policy controller: https://github.com/aws/amazon-network-policy-controller-k8s/blob/f34356594e417edac53b6532ec88f305b3bbb128/internal/controllers/policy_controller.go#L45